cmds/core/yes/yes.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
+	args := os.Args
```

Comment:
I think like Ron said earlier, using flag.Args() would be the best option here because it would take care of a lot of checks for you and all you'd have to do is deal with the actual arguments being passed in
So you should be able to call `flag.Parse()` and then use `flag.Args()` to get the arguments - the slice returned should only consist of actual arguments to the yes cmd, see https://gobyexample.com/command-line-flags for an example

Reply:
Yup, I just fixed that :). Thank you!